test/index.js

```diff
@@ -565,3 +565,36 @@ test('approximateSize() encodes start and end', function (t) {
 
   encdown(down).approximateSize(1, 2, noop)
 })
+
+test('can arbitrarily seek', function (t) {
+  t.plan(7)
+
+  var db = encdown(memdown(), { keyEncoding: 'json' })
+
+  var it = db.iterator({ keyAsBuffer: false, valueAsBuffer: false })
+  db.batch([
+    { type: 'put', key: { a: 'a' }, value: 'A' },
+    { type: 'put', key: { b: 'b' }, value: 'B' },
+    { type: 'put', key: { c: 'c' }, value: 'C' },
+    { type: 'put', key: { d: 'd' }, value: 'D' },
+    { type: 'put', key: { e: 'e' }, value: 'E' }
+  ], (err) => {
+    t.error(err, 'no error')
+    it.seek({ c: 'c' })
```

Comment:
Maybe use number keys instead of object keys, which are uncommon and make this test harder to read.

And I think a small unit test or spy would be more valuable here, as we're not interested in whether the underlying iterator seeks to an expected key, just whether our iterator encodes the target. Along the lines of:

```js
var db = encdown({
  iterator: function () {
    return {
      seek: function (target) {
         t.is(..)
         t.end()
      }
    }
  }
}, { keyEncoding: 'json' })

var it = db.iterator(..)
it.seek(..)
```

Reply:
yeah that makes sense - done. 

(I generally integration-test everything - just to be sure :D )